packages/components/src/Enumeration/Items/Item/Item.scss

```diff
@@ -2,14 +2,12 @@ $tc-enumeration-smaller-padding: $padding-smaller !default;
 $tc-enumeration-small-padding: $padding-small !default;
 
 .tc-enumeration-item {
+	display: flex;
 	font-size: 14px;
 	font-weight: 400;
 	line-height: 11px;
 	text-align: left;
-	padding: $tc-enumeration-small-padding;
```

Comment:
As I put a button instead of a simple span (to be able to select an item), the padding is not needed anymore